src/scheduler.ml

```diff
@@ -371,6 +432,66 @@ end = struct
   let init () = Lazy.force init
 end
 
+module Signal_watcher : sig
+  val init : unit -> unit
+end = struct
+
+  let signos = List.map Signal.all ~f:Signal.to_int
+
+  let warning = {|
+
+*************************************************************
+* Press Control+C again quicly to perform an emergency exit *
```

Comment:
s/quicly/quickly/

Reply:
Fixed

Reply:
No it's not??

Reply:
Bah - stupid GH interface. Sorry for noise...